plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/sync/SyncVspCommand.java

```diff
@@ -39,4 +39,21 @@ public String getNuageVspEntity() {
         return _nuageVspEntity;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        SyncVspCommand that = (SyncVspCommand) o;
+
+        if (_nuageVspEntity != null ? !_nuageVspEntity.equals(that._nuageVspEntity) : that._nuageVspEntity != null)
+            return false;
+
+        return true;
+    }
+
+    @Override
+    public int hashCode() {
+        return _nuageVspEntity != null ? _nuageVspEntity.hashCode() : 0;
+    }
```

Comment:
Please implement ``toString()`` to provide debugging state information for debugging purposes.